cocos/spine/lib/spine-core.js

```diff
@@ -7727,438 +97,72 @@ var spine;
         AttachmentType[AttachmentType["Point"] = 5] = "Point";
         AttachmentType[AttachmentType["Clipping"] = 6] = "Clipping";
     })(AttachmentType = spine.AttachmentType || (spine.AttachmentType = {}));
+
+
+    spine.Vector2 = {};
+    spine.MathUtils = {};
+    spine.Color = {};
+    spine.Interpolation = {};
+    spine.Triangulator = {};
+    spine.ConstraintData = {};
+    spine.IkConstraintData = {};
+    spine.PathConstraintData = {};
+    spine.SkeletonBounds = {};
+    spine.Event = {};
+    spine.EventData = {};
+    spine.Attachment = {};
+    spine.VertexAttachment = {};
+    spine.BoundingBoxAttachment = {};
+    spine.ClippingAttachment = {};
+    spine.MeshAttachment = {};
+    spine.PathAttachment = {};
+    spine.PointAttachment = {};
+    spine.RegionAttachment = {};
+    spine.AtlasAttachmentLoader = {};
+    spine.TextureAtlasPage = {};
+    spine.TextureAtlasRegion = {};
+    spine.TextureAtlas = {};
+    spine.PowOut = {};
+    spine.BoneData = {};
+    spine.SlotData = {};
+    spine.Updatable = {};
+    spine.IkConstraint = {};
+    spine.PathConstraint = {};
+    spine.TransformConstraintData = {};
+    spine.TransformConstraint = {};
+    spine.Bone = {};
+    spine.Slot = {};
+    spine.Skin = {};
+    spine.SkinEntry = {};
+    spine.SkeletonClipping = {};
+    spine.SkeletonData = {};
+    spine.TranslateTimeline = {};
+    spine.ScaleTimeline = {};
+    spine.ShearTimeline = {};
+    spine.RotateTimeline = {};
+    spine.ColorTimeline = {};
+    spine.TwoColorTimeline = {};
+    spine.AttachmentTimeline = {};
+    spine.DeformTimeline = {};
+    spine.EventTimeline = {};
+    spine.DrawOrderTimeline = {};
+    spine.IkConstraintTimeline = {};
+    spine.TransformConstraintTimeline = {};
+    spine.PathConstraintPositionTimeline = {};
+    spine.PathConstraintMixTimeline = {};
+    spine.TrackEntry = {};
+    spine.AnimationStateData = {};
+    spine.AnimationState = {};
+    spine.Animation = {};
+    spine.EventQueue = {};
+    spine.AnimationStateListener = {};
+    spine.AnimationStateAdapter = {};
+    spine.Skeleton = {};
+    spine.SkeletonBinary = {};
+    spine.SkeletonJson = {};
+    spine.VertexEffect = {};
+    spine.JitterEffect = {};
+    spine.SwirlEffect = {};
```

Comment:
What's all this declaration for ? Should we put all contents in this file just in `spine-define.js` ?

Reply:
empty class is not useful. some enums type still be defined in here and i don't want change spine-define.ts to a js file.